package.json

```diff
@@ -45,6 +45,7 @@
     "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-standard": "^4.0.1",
     "eslint-plugin-typescript": "^0.14.0",
+    "events": "^3.2.0",
```

Comment:
does webpack automatically detect this module somehow? I'm surprised that just adding it to package.json is enough to have this be added to the preload bundle.

Reply:
Sorry for late answer, I was on vacation. I have not looked into webpack internal that much to be honest. Most migration guides state that you should include what you use, so I just did that, similarly to original PR adding process and buffer.